src/main/resources/static/css/styles.css

```
@@ -6,68 +6,232 @@
     --text: #1e293b;
     --border: #e2e8f0;
 }
+
 * { box-sizing: border-box; margin: 0; padding: 0; }
-body { font-family: 'Inter', sans-serif; background: var(--bg); color: var(--text); }
+
+body {
+    font-family: 'Inter', sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)